.github/workflows/build.yml

```diff
@@ -162,17 +115,38 @@ jobs:
           retry_wait_seconds: 10
           command: docker pull docker.pkg.github.com/apache/incubator-nuttx-testing/nuttx-ci-linux
 
-      - name: Export NuttX Repo SHA
-        run:  echo "::set-env name=nuttx_sha::`git -C sources/nuttx rev-parse HEAD`"
-      - name: Refresh Git Credentials
+      - uses: actions/download-artifact@v2
+        with:
+          name: gitrefs
+          path: gitrefs-rt/
```

Comment:
should we directly donwload to current the directory?

Reply:
Sounds good.